client/web/src/enterprise/insights/components/insights-view-grid/components/insight-context-menu/InsightContextMenu.tsx

```diff
@@ -40,6 +40,7 @@ export const InsightContextMenu: React.FunctionComponent<InsightCardMenuProps> =
                     <MenuButton
                         data-testid="InsightContextMenuButton"
                         className={classnames(menuButtonClassName, 'btn btn-outline p-1', styles.button)}
+                        aria-label="Insight context menu"
```

Comment:
The "menu" here would sound redundant to the screen reader as the role is announced as "menu" too, maybe "Insight options"?
```suggestion
                        aria-label="Insight options"
```
Similarly to how you removed the "button" suffix from the label in the other file.

Reply:
Oh good catch you are totally right: "Insight context menu, menu popup button"